File: src/A/UVA10110.java
package A;

import java.util.Comparator;
import java.util.Scanner;

public class UVA10110 {
    public static void main(String[] args) {
        Comparator.comparing()
        Scanner scan=new Scanner(System.in);
        int n=scan.nextInt();
        while (n!=0){
            System.out.println(solve(n)?"yes":"no");
            n=scan.nextInt();
        }
    }
    public static boolean solve(int n) {
        int x=(int)Math.sqrt(n);

        return x*x==n;
    }
}

File: src/A/UVA 10110.kt
package A
/*
3
6241
8191
 */

fun main() {
    var n= readLine()!!.toInt()
    while (n!=0){
        println(if (solve(n))"Yes" else "No")
        n= readLine()!!.toInt()
    }
}

fun solve(n: Int): Boolean {
    val x = Math.sqrt(n.toDouble()).toInt()
    return x * x == n
}
